posixlib/src/main/scala/scala/scalanative/posix/getopt.scala

```diff
@@ -4,7 +4,8 @@ package posix
 import scalanative.unsafe._
 
 @deprecated(
-  "getopt is no longer part of POSIX 2018 and will be removed. Use unistd instead"
+  "getopt is no longer part of POSIX 2018 and will be removed. Use unistd instead.",
+  "0.5.0"
```

Comment:
Thanks. This is much informative.

Reply:
Running the next leg of the relay; building on your efforts.